src/mod/server.mod/tclserv.c

```diff
@@ -599,6 +603,10 @@ static int tcl_server STDVAR {
       Tcl_SetObjResult(irp, server);
     }
   } else if (!strcmp(argv[1], "remove")) {
+    if (argc < 3) {
+      Tcl_SetResult(irp, "Subcommand remove needs at least a server host", TCL_STATIC);
```

Comment:
I would use the error message `wrong # args: should be "server add host ?port? ?password?"`

Reply:
done